docs/advanced.rst

```diff
@@ -428,4 +445,4 @@ This may be useful when you want to construct timelines and need to get only the
     new_record, old_record = p.history.all()
     delta = new_record.diff_against(old_record)
     for change in delta.changes:
-        print("{} changed from {} to {}".format(change.field, change.old, change.new))
\ No newline at end of file
+        print("{} changed from {} to {}".format(change.field, change.old, change.new))
```

Comment:
I don't see a diff here? Spacing?

Reply:
GitHub adds a blank line at the end. I'm going to fix it.

Reply:
No, it doesn't. I'm afraid I don't know how to fix this.

Reply:
No worries. Merging. Thanks!